src/components/main/main.jsx

```diff
@@ -141,6 +153,17 @@ function Main1() {
   function createGameHandler(gameName) {
     let r = Math.random().toString(36).substring(7);
     console.log(r);
+    if(playerName==""){
+      setplayerNameMessage("enter valid player name");
+    }
+    else{
+      setplayerNameMessage("");
+    }
+    if(gameName==""){
+    setgameNameMessage("enter valid game key");
+    }
+    else{
```

Comment:
Control will reach here even if playerName is null because we are not returning from the function after the if condition for playerName above. I think it'll set the error msg to enter valid player name, but still will create a game.